test/powershell/Modules/Microsoft.PowerShell.Utility/Get-FileHash.Tests.ps1

```diff
@@ -3,8 +3,10 @@
 Describe "Get-FileHash" -Tags "CI" {
 
     BeforeAll {
-        $testDocument = Join-Path -Path $PSScriptRoot -ChildPath assets testablescript.ps1
+        $testDocument = Join-Path -Path $TestDrive -ChildPath "hashable.txt"
         Write-Host $testDocument
```

Comment:
We shouldn't have a `Write-Host` here, seems like a leftover when this was written for debugging purposes.  Just delete this line.